src/components/TopStory.vue

```diff
@@ -30,84 +33,32 @@
 </script>
 
 <template>
-  <article class="module top-story">
-    <h2>TOP STORY</h2>
-    <img class="article-logo" :src="article_1_logo" alt="llamas">
-    <img :src="article_1['og:image']" alt="ham">
-    <h3>{{ article_1['og:title'] }}</h3>
-    <p>{{ article_1_body }}</p>
-    <a :href="article_1_url"><button class="button read-more" type="button">
-      Read More
-    </button></a>
+  <article class="card card-single card_top-story">
+    <header class="card-header">
+      <h2 class="card-title">Top Story</h2><span class="devtip"> // .card-single .card_top-story</span>
+    </header>
+    <div class="card-content">
+      <article class="top-story">
+        <header class="article-header">
+          <img :src="article_1['og:image']" alt="ham">
+          <div class="article-meta">
+            <span class="item-source">
+              <!-- todo: allow for image or text fallback -->
+              <img class="article-logo" :src="article_1_logo" alt="llamas">
+            </span>
+            date, author, readtime, etc.
+          </div>
+        </header>
+        <h3 class="item-title">{{ article_1['og:title'] }}</h3>
+        <div class="item-body">{{ article_1_body }}</div>
+       </article>
+    </div>
+
+    <CardFooter />
+
   </article>
 </template>
 
 <style scoped lang="scss">
-  button.button {
-    border-radius: 16px;
-    background-color: blue;
-    border: 0;
-    text-align: center;
-    background-color: $background-light;
-    padding: 0 20px;
-    line-height: 2.5;
-    font-weight: bold;
-    position: absolute;
-    bottom: 1rem;
-    right: 1rem;
-    opacity: 75%;
-    cursor: pointer;
-    &:hover {
-      opacity: 100%;
-      transition: all 0.5s ease-in-out;
-      filter: drop-shadow(0 0 15px $background-dark);
-    }
-  }
-  article.top-story {
```

Comment:
I think eventually this should be moved back to the component level since its component-level scss. but fine to move out for now